src/components/RepoSubmission.tsx

```diff
@@ -67,25 +67,25 @@ const RepoSubmission = ({ isFormOpen, handleFormOpen }: RepoSubmissionProps): JS
   return (
     <div
       ref={submissionRef}
-      className="fixed top-[60px] right-[120px] flex items-end flex-col gap-[10px] submission-form z-10"
+      className="fixed top-[3.75rem] right-[7.5rem] flex items-end flex-col gap-2.5 submission-form z-10"
```

Comment:
why the rems?

Reply:
Tailwind does not have 60px, and I said to translate them in rem. but still custom. But I put them in standard classes from a tailwind.

Reply:
with standard classes 
<img width="550" alt="image" src="https://user-images.githubusercontent.com/37635083/189475908-2beaf5a0-cada-4d64-a87a-549b9b392d45.png">

Reply:
remove "left" entirely and use "right" instead - it's supposed to be 5 pixels from the right, fixing that bug enables tailwind sizing

Reply:
It is ok like that 
![image](https://user-images.githubusercontent.com/37635083/189481129-512f6fe4-9a0d-45c3-b15a-591d47e87e16.png)